hs_core/migrations/0001_initial.py

```diff
@@ -235,7 +235,9 @@ class Migration(migrations.Migration):
             fields=[
                 ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
                 ('object_id', models.PositiveIntegerField()),
-                ('type', models.CharField(max_length=100, choices=[(b'isPartOf', b'Part Of'), (b'isExecutedBy', b'Executed By'), (b'isCreatedBy', b'Created By'), (b'isVersionOf', b'Version Of'), (b'isDataFor', b'Data For'), (b'cites', b'Cites')])),
+                ('type', models.CharField(max_length=100, choices=[(b'isHostedBy', b'Hosted By'), (b'isCopiedFrom', b'Copied From'), (b'isPartOf', b'Part Of'),
```

Comment:
@pkdash No, they are not. In fact, Django makemigrations does not result in any change in migration files. I added two new field choices in just for consistency. They are not really needed since it works just fine without changing or adding any migration files. Should we add them or not? Either works, but just to be safer and consistent with other field choices, I thought it is better to add them in given that Django makemigrations does not add any migration files in this case (i.e., addition of two new choices for the type field). What do you think?